Doc/library/random.rst

```diff
@@ -412,6 +412,31 @@ Alternative Generator
       ``None``, :class:`int`, :class:`float`, :class:`str`,
       :class:`bytes`, or :class:`bytearray`.
 
+   .. method:: Random.seed()
+
+      Override this method in subclasses to customise the :meth:`random.seed`
+      behaviour of :class:`Random` instances.
+
+   .. method:: Random.getstate()
+
+      Override this method in subclasses to customise the :meth:`random.getstate`
+      behaviour of :class:`Random` instances.
+
+   .. method:: Random.setstate()
```

Comment:
Also not sure about this signature: https://docs.python.org/3/library/random.html#random.setstate